go-controller/hack/update-modelgen.sh

```diff
@@ -0,0 +1,19 @@
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# generate ovsdb bindings
+if  ! ( command -v modelgen > /dev/null ); then
+  echo "modelgen not found, installing github.com/ovn-org/libovsdb/cmd/modelgen"
+  olddir="${PWD}"
+  builddir="$(mktemp -d)"
+  cd "${builddir}"
+  GO111MODULE=on go install github.com/ovn-org/libovsdb/cmd/modelgen@a4f2602f585a3c2995a0abea7010b333631341d9
```

Comment:
Maybe better to use a variable here, so we can pass it in ro override, or use a default value `4f2602f585a3c2995a0abea7010b333631341d9` ?

Reply:
thanks for review Fla! hmm not sure I follow, is having a variable making any difference here? I think no...we'd not really use that value any differently since variable would have to be called with a hardcoded value from Makefile right? and some has to change the hash there? LMK if what's what you had in mind?

Having said I should at least have some docs outlining this information to anyone who does the modelclient generation to go and change this hash as well to the latest like you had mentioned in the meeting.

Reply:
I have added extra comments and docs to ensure folks are aware, see if that is fine?

Reply:
yes, this is perfectly fine!

Reply:
Thank you for reviews!! Resolving this comment.